src/coreclr/jit/codegenloongarch64.cpp

```diff
@@ -9238,15 +9332,16 @@ void CodeGen::genFnPrologCalleeRegArgs()
                             tmp_offset = base;
                             tmp_reg    = REG_R21;
                             GetEmitter()->emitIns_I_la(EA_PTRSIZE, REG_R21, base);
-                            // NOTE: `REG_R21` will be used within `emitIns_S_R`.
+                            // NOTE: `REG_R21` will be used within `emitIns_S_R` which including the large offset imm,
+                            // this is optimized for generating reduntant large offset imm of the store instructions.
```

Comment:
```
 // NOTE: `REG_R21` will be used within `emitIns_S_R` which including the large offset imm,
// this is optimized for generating reduntant large offset imm of the store instructions.
```

Can you please reword this comment too? Are you trying to say that `emitIns_S_R` takes care of eliminating redundant store, if exists, using `IsRedundantLdStr()`. If yes, then I don't think that need to be mentioned here.

Reply:
I had rewritten this.
thanks